CHANGELOG.unreleased.md

```diff
@@ -7,6 +7,8 @@
 
 > Users must be able to say: “Nice enhancement, I'm eager to test it”
 
+- [Vm] Display the full driver version in the general and advanced tab instead of `major.minor`
```

Comment:
`Vm` -> `VM` because it's an acronym.  And please add PR and issue links.

Reply:
@badrAZ A suggested change can help :slightly_smiling_face:

Reply:
```suggestion
- [VM] Display the full driver version in the general and advanced tab instead of `major.minor` [#5680](https://github.com/vatesfr/xen-orchestra/issues/5680) (PR [#5691](https://github.com/vatesfr/xen-orchestra/pull/5691))
```